src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -68,11 +68,35 @@ const Inventory = ({
           {}
         )
       )?.data.data;
-      const issues = pathways.map((rec) => ({
-        id: `advisor:${rec.rule_id}`,
-        description: rec.description,
-      }));
-      return { issues, systems: selected };
+
+      const systems = (
+        await Get(
+          `${BASE_URL}/pathway/${encodeURI(pathway.slug)}/reports/`,
+          {},
+          {}
+        )
+      )?.data.rules;
+
+      let issues = [];
+      pathways.forEach((rec) => {
+        let filteredSystems = [];
+
+        systems[rec.rule_id].forEach((system) => {
```

Comment:
I am wondering if systems array has always rule with associated rule_id, if it is not there might be an edge case. It will complain about undefined index/value.

Reply:
so just to be save, I should change it to `systems[rec.rule_id]?.forEach((system)` ? But it should always contain `rule_id`